lib/mal_import.rb

```diff
@@ -154,6 +154,14 @@ def to_h
   end
 
   private
+  def character_image(img)
+    img = img.gsub("t.jpg", ".jpg")
+    if img.include?("questionmark")
+      nil
+    else
+      URI(img)
+    end
```

Comment:
you can change this `if` to:
```ruby
URI(img) unless img.include?("questionmark")
```
it automatically add `nil` if the last line is not executed :)